### DISCUSSION TITLE
shortcuts for showing / hiding hidden files (solved)

### DISCUSSION THREAD

**Discussion by maiemi** [mucommander/mucommander#1161] [Ideas]
first I will thank for that great tool and the hard work behind!
I gently ask for a way to configure a shortkey to let appear or hide hidden files in the active window of muCommander. I studied the helpfile(s) but did not found any hint to realize this. There are some shortkeys explained, but no one for hiding or appearing hidden files (and folders). I know the way to show or hide hidden files (and folders) via the settings menu, but a shortkey would be practical.
Maybe someone else will find this idea nice, too :)
regards
maiemi

**Comment by ghost** [comment]
How about Alt+Shift-H

**Reply by maiemi** [reply]
> How about Alt+Shift-H

that works! Thank you
